subsys/bluetooth/controller/ll_sw/ull_llcp.c

```diff
@@ -467,6 +514,23 @@ static void pdu_encode_start_enc_rsp(struct pdu_data *pdu)
 	/* TODO(thoh): Fill in PDU with correct data */
 }
 
+static void pdu_encode_reject_ind(struct pdu_data *pdu, u8_t error_code)
```

Comment:
IMHO we should move the encode routines to a separate file; giving an even more clear separation of the structure (i.e. how we send/receive control procedures) and the content (i.e. what we are sending). 
Also this module will become quite large otherwise

Reply:
We could also divide the code into procedure modules, i.e. ull_llcp_enc.c, ull_llcp_phy.c etc...

Reply:
OK, not a blocker for this PR; we'll refactor later